source/_patterns/01-molecules/components/carousel-item.yaml

```diff
@@ -24,6 +24,9 @@ properties:
   title:
```

Comment:
I can't view this molecule standalone, just have a blank page. Should I be able to?

Edit: Actually I can't seem to get any of the Carousel molecules to display be selecting them from the `molecules->components` menu. Other molecules work, so I assume this should too?

Reply:
So I just tested this locally without your changes and it's still broken, so not a regression as a result of your change. I'll raise a separate issue to cover fixing them.